scripts/mbedtls_dev/__init__.py

```diff
@@ -1,3 +1,7 @@
 # This file needs to exist to make mbedtls_dev a package.
 # Among other things, this allows modules in this directory to make
 # relative imports.
```

Comment:
My information may be outdated. 

When you do `from . import module`  module is not imported from the current directory, but  from the top level of the package. `.` indicated the root of the package.

Overriding `__all__` was a surefire way to control what is provided by the package, and then relative imports could work. 

To be frank I did not test that specific use-case.

One other note, this globbing functionality does not have to be be here if only one other libary is using it. If we only want to alias this library, from the test scripts, it could go into the `mbedtls_test_dev/__init__.py`

If though we expect to have libraries in multiple locations, like framework, tests, or ci, then it would be needed at this file, since it is where the actual code base resides in.

e.g


### single alias

~~~~~~
mbedtls_dev/__init.py__


mbedtls_test_dev/__init.py__
  list = mbedtls_dev.glob()
  from mbedtls_dev import list 

~~~~~~

### Glob once and re-use

~~~~~~
mbedtls_dev/__init.py__
  glob()


mbedtls_test_dev/__init.py__
  from mbedtls_dev import *

mbedtls_framework_dev/__init.py__
  from mbedtls_dev import *
~~~~~~